Ramanujan_joel.java

```diff
@@ -0,0 +1,52 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by Fernflower decompiler)
+//
+
+public class Ramanujan {
+  public Ramanujan() {
+  }
+
+  private static boolean isWholeNumber(double error, double n) {
```

Comment:
can you remove this function?

Reply:
Yes,  because I only use it in one place it probably shouldn't be a separate function.